biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -534,6 +534,7 @@ subroutine nan_cohort(cc_p)
     currentCohort%ddbhdt             = nan ! time derivative of dbh 
 
     ! FIRE
+    currentCohort%sh                    = nan ! scorch height affecting crown (m)
```

Comment:
could we make this variable a little more verbose? When I see sh without context, my mind goes to specific humidity.

Reply:
yep. will update now.